parquet/src/main/java/org/apache/iceberg/parquet/ParquetDictionaryRowGroupFilter.java

```diff
@@ -406,6 +406,8 @@ public Boolean or(Boolean leftResult, Boolean rightResult) {
 
       for (int i = 0; i <= dict.getMaxId(); i++) {
         switch (col.getPrimitiveType().getPrimitiveTypeName()) {
+          case FIXED_LEN_BYTE_ARRAY: dictSet.add((T) conversion.apply(dict.decodeToBinary(i)));
+            break;
```

Comment:
I wanted to write the following but Checkstyle failed on it because of indention. I guess, it is/was a [Checkstyle bug](https://github.com/checkstyle/checkstyle/issues/341). I don't have experience on gradle so I am not sure how to fix this in Iceberg configuration. Probably, we need to upgrade Checkstyle somehow.

Anyway, both the current and this other one is correct but I would prefer the latter.
```suggestion
          case FIXED_LEN_BYTE_ARRAY:  // Same as BINARY
```